frontend/styles/footer.css

```diff
@@ -0,0 +1,167 @@
+* 
+{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+footer
+{
+    border-top: 1.5px solid rgb(180, 180, 180);
+}
+
+.footer__logo-box
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 5rem;
+}
+
+.row
```

Comment:
since many people are working on the project so make sure className is unique throughout.
Ex- .footer__row